griptape/utils/chat.py

```diff
@@ -9,16 +9,25 @@
 @define(frozen=True)
 class Chat:
     structure: Structure = field()
+    exit_keywords: list[str] = field(default=["exit"], kw_only=True)
+    exiting_text: str = field(default="exiting...", kw_only=True)
+    processing_text: str = field(default="processing...", kw_only=True)
+    intro_text: Optional[str] = field(default=None, kw_only=True)
+    prompt_prefix: str = field(default="Q: ", kw_only=True)
+    response_prefix: str = field(default="A: ", kw_only=True)
+    output_fn: Callable[[str], None] = field(default=Factory(lambda: print), kw_only=True)
```

Comment:
We use `func` in `SnowflakeSqlDriver.connection_func` and `fn` in `LocalVectorStoreDriver.relatedness_fn`. Some other callables look like regular methods (i.e., verb, no suffix).

Not sure what to do here. Just go with `fn` and address the rest later?